public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -46,7 +46,7 @@ const mapDispatchToProps = {
 
 const connector = connect(mapStateToProps, mapDispatchToProps);
 
-const ServiceAccountsListPage = ({
+const ServiceAccountsListPage: FC<Props> = ({
```

Comment:
We're trying to move away from using `React.FC`, the changed code is the recommended approach.

Reply:
Just curious why this is? 🤔

Reply:
I think mostly because of typing issues that FC<Props> may introduce.